databases/week2/MySQl-Homework-Week2 part4.sql

```diff
@@ -0,0 +1,72 @@
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION';
+
+-- -----------------------------------------------------
+-- Schema farm
+-- -----------------------------------------------------
+
+-- -----------------------------------------------------
+-- Schema farm
+-- -----------------------------------------------------
+CREATE SCHEMA IF NOT EXISTS `farm` DEFAULT CHARACTER SET utf8 ;
+USE `farm` ;
+
+-- -----------------------------------------------------
+-- Table `farm`.`farmers`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `farm`.`farmers` (
+  `id` VARCHAR(45) NOT NULL,
+  `farmer_id` INT NOT NULL,
+  `farmer_name` VARCHAR(45) NOT NULL,
+  `farmer_address` VARCHAR(45) NOT NULL,
+  `farmer_mobile` INT(10) NOT NULL,
+  `animal_id` INT NULL,
```

Comment:
remove from this table and put it in an intermediary table which connects the farmer to many animal. This one allows you to assign only one animal to each farmer.

Reply:
schema changed